modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -109,14 +107,27 @@ valid_lft forever preferred_lft forever
 link/ether 62:b1:b5:c8:fb:7a brd ff:ff:ff:ff:ff:ff
 inet 10.244.1.122/24 brd 10.244.1.255 scope global eth0
 valid_lft forever preferred_lft forever
-4: bond0: <BROADCAST,MULTICAST,UP,LOWER_UP400> mtu 1500 qdisc noqueue state UP qlen 1000
-link/ether 9e:23:69:42:fb:8a brd ff:ff:ff:ff:ff:ff
+4: net3: <BROADCAST,MULTICAST,UP,LOWER_UP400> mtu 1500 qdisc noqueue state UP qlen 1000
+link/ether 9e:23:69:42:fb:8a brd ff:ff:ff:ff:ff:ff <1>
 inet 10.56.217.66/24 scope global bond0
 valid_lft forever preferred_lft forever
 43: net1: <BROADCAST,MULTICAST,UP,LOWER_UP800> mtu 1500 qdisc mq master bond0 state UP qlen 1000
-link/ether 9e:23:69:42:fb:8a brd ff:ff:ff:ff:ff:ff <1>
-44: net2: <BROADCAST,MULTICAST,UP,LOWER_UP800> mtu 1500 qdisc mq master bond0 state UP qlen 1000
 link/ether 9e:23:69:42:fb:8a brd ff:ff:ff:ff:ff:ff <2>
+44: net2: <BROADCAST,MULTICAST,UP,LOWER_UP800> mtu 1500 qdisc mq master bond0 state UP qlen 1000
+link/ether 9e:23:69:42:fb:8a brd ff:ff:ff:ff:ff:ff <3>
+----
+<1> The bond interface is automatically named `net3`. To set a specific interface name add `@name` suffix to the pod’s `k8s.v1.cni.cncf.io/networks` annotation.
+<2> The `net1` interface is based on an SR-IOV virtual function.
+<3> The `net2` interface is based on an SR-IOV virtual function.
++
+[NOTE]
```

Comment:
IMO, I think it might be better to add this as an optional step, rather than a note. This would mean starting a numbered list at line 96.